jobs/geo/write_building_footprints.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from jobs.utils.snowflake import snowflake_connection_from_environment
+
+
+def write_building_footprints(conn):
+    """Grab Microsoft state building footprint data for California from Snowflake and write to an S3 bucket."""
+    import geopandas
+    import shapely
+
+    sql_transform = """
+    alter session set GEOGRAPHY_OUTPUT_FORMAT='WKB';
+    """
+    conn.cursor().execute(sql_transform)
+
+    sql_table = """
+    SELECT *
+    FROM ANALYTICS_DEV.ANALYTICS.GEO_REFERENCE__BUILDING_FOOTPRINTS_WITH_BLOCKS
```

Comment:
This is likely a big file! Does your laptop struggle with it at all? It might be better to query per county, and process them in serial.

Reply:
I haven't tried yet. I knew the intention was to get the entire dataset so I removed the limit when I was done testing. This feels like a suggestion and not a requirement for the code to work or us to get our desired output, is my understanding correct?

Reply:
I might expect it to crash your computer. I suggest trying it. Even if it succeeds, I still expect it to work better processing them in serial.

Remember, this dataset is large to the point of being difficult (but not impossible) for a single machine to process, and we've just made it bigger.